### ISSUE TITLE
try to work on ESP8266 arduino

### ISSUE THREAD

**Issue by asetyde** [olewolf/DHT_nonblocking#2]
Hi , now i will try to adapt your library , that is good choice for non-blocking feature , do you can help me ?

**Comment by olewolf** [comment]
Glad to hear you find my library useful. What is your specific question?

**Comment by asetyde** [comment]
first things to improve it is adapt this option in library prop.


from architectures=avr to architectures=*  (i use sometimes visualstudio with arduino micro , this option is needed)

**Comment by asetyde** [comment]
this is git of adafruit , it uses from most programmar of esp
https://github.com/adafruit/DHT-sensor-library/

**Comment by asetyde** [comment]
this is esp git
https://github.com/esp8266/Arduino

**Comment by olewolf** [comment]
I'm sorry, but changing my code to indicate that "architectures=*" would imply that it works on all architectures. I only expect it to work on a 16 MHz "vanilla" Arduino, which probably excludes several architectures. I'll try and look into the library.properties file to see which Arduino types are likely to be supported, but '*' doesn't seem likely.

**Comment by asetyde** [comment]
yes , it's true . i will try in these days your library and if all go as i think we can add  

architectures=avr,esp8266

 arch ok ?

**Comment by asetyde** [comment]
i 've problem with visual micro to import library .. when i risolve it and i go to test it

**Comment by asetyde** [comment]
not read value , i don't know why

**Comment by kbickham** [comment]
I was about to go through and start removing delays, but checked first. Thank you for doing this. I actually, figured out the data transfer protocol reading your comments. 28 usex low, 70usec high on the logic high pulses. These things are good sensors, but the original library is a pain in terms of timing if you have a very complex system to integraste.
